deploy/bav-spec.yaml

```diff
@@ -0,0 +1,197 @@
+openapi: 3.0.1
+info:
+  title:
+     Fn::Sub: "${AWS::StackName} - BAV Credential Issuer Private API"
+  version: "1.0"
+  description: >-
+    Bank Account Verification CRI - Frontend and Backend API specification
+  contact:
+    name: Darren Dunford
+    email: darren.dunford@digital.cabinet-office.gov.uk
```

Comment:
I wonder if this should be @mudassar-jafer going forward?

Reply:
good shout - I've checked with him and he is happy for us to do that so have made the change